algoliasearch-core/src/main/java/com/algolia/search/util/AlgoliaUtils.java

```diff
@@ -93,30 +89,6 @@ public static Boolean isNullOrEmptyWhiteSpace(final String stringToCheck) {
             + "must have an objectID property or a Jackson annotation @JsonProperty(\"objectID\")");
   }
 
-  public static CompletableFuture<HttpResponse> toCompletableFuture(
```

Comment:
This inlining should be done in a separate, dedicated commit: it is actually unrelated to the move to multi-module

Reply:
It is! This method takes an `org.apache.http.HttpResponse` as an input, as we wanted to have `algoliasearch-core` completely independent from any http implementetion we had to move this helper in `ApacheRequester`.

Reply:
> It is!
It is not: it is possible to inline this helper in ApacheHttpRequester right now - without splitting the code into multi-module. For sure, it must be done for splitting cleanly. But it can be done beforehand.